index.html

```diff
@@ -123,24 +152,24 @@ <h2>About Myself</h2>
         <button>Get my resume</button>
```

Comment:
- [ ] [OPTIONAL] i would suggest that you add a `cursor: pointer;` style to all of your button to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

Before            |  After
:-------------------------:|:-------------------------:
![G1](https://user-images.githubusercontent.com/89282221/186756877-f7657a5c-d2ee-4421-ac8d-b34d524495eb.gif)  |  ![G2](https://user-images.githubusercontent.com/89282221/186756884-438d4a24-ddd3-40bc-87aa-73ced259cc57.gif)